### DISCUSSION TITLE
IOS device mirroring on Windows

### DISCUSSION THREAD

**Discussion by hackerish1** [NetrisTV/ws-scrcpy#200] [Q&A]
How do i connect an iphone to windows machine and configure to mirror the screen using ws-scrcpy_

Tried connecting my iphone to my windows laptop (8gb ryzen 5). and started npm 
Error: spawn D:\D Drive\B2BTesters\Dev\ws-scrcpy\node_modules\ios-device-lib\bin\win32\x64ios-device-lib ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:480:16)
    at processTicksAndRejections (node:internal/process/task_queues:81:21)
got above error. can someone help me out with this. thanks in advance

**Comment by drauggres** [comment]
Hi. Even if you fix `ENOENT`, it sill will be impossible to use mirroring:
- [ws-qvh](https://github.com/NetrisTV/ws-qvh) does not work on Windows
- there is no xcode for Windows (i.e. you would be able to start WebDriverAgent to control ios-device).

A possible solution is to implement #195.
